index.html

```diff
@@ -30,26 +30,172 @@ <h1>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <button>See project</button>
+        </article>
+        <article class="work workHeight">
+          <img src="assets/Snapshoot Portfolio-4.svg" alt="" />
+          <h3>Multi-Post Stories</h3>
+          <ul class="skill-info">
+            <li>CANOPY</li>
+            <li><img src="assets/Counter.png" alt="" /></li>
+            <li>Back End Dev</li>
+            <li><img src="assets/Counter.png" alt="" /></li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="work-lang">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <button>See project</button>
+        </article>
+      </section>
+
+      <section class="about-section">
+        <article>
+          <h2>About Myself</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website. Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+          <div class="contactTitle"><a href="#">LET’S CONNECT</a></div>
+          <ul class="lets-connect">
+            <li>
+              <a href="#"><img src="assets/Vector.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="assets/Linkedin icon-2.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="assets/Vector-2.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="assets/Vector-3.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="assets/Vector-4.svg" alt="" /></a>
+            </li>
+          </ul>
+          <button class="get-resume-btn">Get my resume</button>
+          <div class="lang-title">
+            <span>Languages</span>
+            <img src="assets/Union.png" alt="" />
+          </div>
+          <ul class="languages">
+            <li>
+              <img src="assets/Ellipse 1.png" alt="" />
+              <span class="js-html-css">JavaScript</span>
+            </li>
+            <li>
+              <img src="assets/Ellipse 1-4.png" alt="" />
+              <span class="js-html-css">HTML</span>
+            </li>
+            <li>
+              <img src="assets/Ellipse 1-5.png" alt="" />
+              <span class="js-html-css">CSS</span>
+            </li>
+          </ul>
+          <div class="frame-title">
+            <span>Frameworks</span>
+            <img src="assets/Union2.png" alt="" />
+          </div>
+          <hr />
+          <div class="skill-title">
+            <span>Skills</span>
+            <img src="assets/Union2.png" alt="" />
+          </div>
```

Comment:
- [ ] KIndly make sure `Languages`, `Frameworks`, and `Skills` are wrapped into the `ul` tag and each one is wrapped into the `li` tag in order to follow best HTML practices and meet the project's requirements a well.